standard_forms/purchase-order/latest/convert_data_js/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "convert_data_js",
+  "version": "1.0.0",
+  "main": "convert_data.js",
+  "scripts": {
+    "build": "webpack"
+  },
```

Comment:
would be good to add test script: 
```
    "test": "jest"
```

Can we please try this out and get the test to run, I think it'll require a few more dev dependencies